CGPs/cgp-0063.md

```diff
@@ -1,51 +1,78 @@
 ---
 cgp: 63
-title: Increase gas price
-date-created: <date created on, in ISO 8601 (yyyy-mm-dd) format>
-author: <a list of the author's or authors' name(s) and/or username(s), or name(s) and email(s), e.g. (use with the parentheses or triangular brackets): FirstName LastName (@GitHubUsername), FirstName LastName <foo@bar.com>, FirstName (@GitHubUsername) and GitHubUsername (@GitHubUsername)>
-status: <DRAFT | PROPOSED | EXECUTED | EXPIRED | WITHDRAWN>
-discussions-to: <link to discussion on forum.celo.org>
-governance-proposal-id: [if submitted]
-date-executed: [if executed] <date created on, in ISO 8601 (yyyy-mm-dd) format>
+title: Increase minimum gas threshold
+date-created: 2022-11-21
+author: Martin (@martinvol), Mathieu (@prdouct-matt)
+status: DRAFT
+discussions-to: https://forum.celo.org/t/increase-of-minimum-gas-fee/4616
+governance-proposal-id: NA
+date-executed: NA
 ---
 ## Overview
 
-CGP - Celo Governance Proposal
+CGP - Celo Governance Proposal 0063
 
-Pick a four digit number subsequent to the latest PR raised and name your file that with the suffic P (for pending).
-Eg. if the latest PR to the CGP folder was [0034], then your PR should be [0035P].
+### Definitions
+Let 
+* gas_price_minimum = the minimal gas price for a given block = a function of (gas_price_minimum of last block, utilization of the last block, gas_price_minimum_(lower_bound))
+* utilization of a block = (gas used in block)  / (gas limit of block)
+* gas_price_minimum_(lower_bound) = a lower bound of the function gas_price_minimum beyond which it can never fall, even if blocks are empty
+* gas_price = the gas price specified by a user for a transaction
+* gas_used = the gas used by a transaction
+* base = gas of a transaction paid to community fund = gas_price_minimum * gas_used
+* tip = gas of a transaction paid to validators = gas_price * gas_used - base
 
-### Status
-- DRAFT = Feedback collection
-- PROPOSED = The governance proposal was submitted to the network
-- EXECUTED = The governance proposal was executed on the network
-- EXPIRED = The governance proposal expired
-- WITHDRAWN = The governance proposal is withdrawn before being submitted on the network
+### Current Situation
+Currently, gas pricing on Celo roughly works as follows: 
+* Every block a gas_price_minimum is defined (accessible through the GasPriceMinimum smart contract). If the block is full (measured in gas), gas_price_minimum increases, if the block is empty, gas_price_minimum decreases (until gas_price_minimum_(lower_bound) is reached). These increases and decreases can be exponential, and thus the value of gas_price_minimum can change quickly.
+* In any given block, only transactions specifying a gas_price higher or equal than gas_price_minimum are valid
+* The gas paid by users is split between validators (tip) and the community fund (base)
```

Comment:
and when the price is at minimum, in reality the majority goes to the community fund.

Reply:
at minimum, I believe 100% goes to community fund? I've added equations for tip and base which makes this clear.